docker/README.md

```diff
@@ -1,3 +1,6 @@
+### Deprecation notice:
```

Comment:
I would suggest moving this down. This is the main docker README and might be confusing to customers who might not have any idea about neuron

Reply:
In my experience I have seen these kind of deprecation notices put at the top. Also provided link to the specific dockerfile to not confuse users. I don't have a strong opinion on this though. Do you suggest putting it at the end?

Reply:
Not a big deal. I was just concerned about customers being confused about neuron devices given we don't have any other documentation for it.